swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
As part of this work can we take this opportunity and move the config file out of the root of the repo? I'd like to have it under the eng folder.

Reply:
Yes we could make it!

Reply:
Let's do it one by one OK? First .NET SDK? Do you have any preference for the config file's name?

Reply:
Can we name this `swagger_to_sdk_config_csharp_v3.json` that way we name it after the codegenerator instead of a code name for our libraries.

Reply:
Can we also use similar naming in the specs repo for the new configuration you are setting up?

Also I'm fine with moving this under the `eng` folder later but I figured it would be easier to do while initially setting things up.